### DISCUSSION TITLE
RNA-Seq by example chapter 5: combining count matrix

### DISCUSSION THREAD

**Discussion by nirjharaloy** [biostars/biostar-handbook-issues#190] [General]
cat ids | combine out1 > counts1.txt
**/usr/bin/env: 'python': No such file or directory**

I'm stuck here. I think python wasn't integrated correctly.

**Comment by nirjharaloy** [comment]
Please address this issue. Do I need to install python in Linux?

**Comment by ialbert** [comment]
perhaps your environment is not activated

     conda activate bioinfo

you can run it through python itself

    cat ids | python combine out1 > counts1.txt

basically, it is telling you don't have python, but then you could not have run any tool like the doctor.py etc, it is quite puzzling

when you installed your software python was the first thing installed

**Comment by nirjharaloy** [comment]
Goodness! I was doing it in salmon environment. Please edit this section in the book and add 
**#revert back to bioinfo from salmon to run these codes**
conda activate bioinfo

Thanks. I found that in some parts of the book the switching of environments were not clearly mentioned. I'll update if I find more situations like this. Thank you very much!
